### PR TITLE
Add Voice Broadcast labs setting and composer button

### PR DESCRIPTION
**Pull request by weeman1337**
Start of https://github.com/vector-im/element-web/issues/23282 „Voice Broadcasts“

* Labs setting
* Composer button

[Designs](https://www.figma.com/file/5zlCPsSOJELBzIiQdHC2LV/Voice-Broadcasting)

## Checklist

* [x] Tests written for new code (and old code if feasible)
* [x] Linter and other CI checks pass
* [x] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: src/components/views/rooms/MessageComposer.tsx [hunk 1]
```diff
@@ -153,7 +157,7 @@ export default class MessageComposer extends React.Component<IProps, IState> {
     public componentDidMount() {
         this.dispatcherRef = dis.register(this.onAction);
         this.waitForOwnMember();
-        UIStore.instance.trackElementDimensions(`MessageComposer${this.instanceId}`, this.ref.current);
+        UIStore.instance.trackElementDimensions(`MessageComposer${this.instanceId}`, this.ref.current!);
```
Comment (kerryarchibald) [inline src/components/views/rooms/MessageComposer.tsx:160]: TIL about the non-null assertion operator, cool

File: src/settings/Settings.tsx [hunk 2]
```diff
@@ -101,6 +101,10 @@ export enum LabGroup {
     Developer,
 }
 
+export const FEATURES = {
+    VOICE_BROADCAST: "voice_broadcast",
```
Comment (kerryarchibald) [inline src/settings/Settings.tsx:105]: maybe should still use the `feature_` prefix for the value to match the existing feature settings? also could be nice to use an enum

File: test/components/views/rooms/MessageComposer-test.tsx [hunk 3]
```diff
@@ -68,10 +77,261 @@ describe("MessageComposer", () => {
...
+                dis.dispatch({
+                    action: "reply_to_event",
+                    context: (wrapper.instance as unknown as MessageComposer).context,
+                });
+                wrapper.update();
+
+                setTimeout(() => {
+                    expect(resizeNotifier.notifyTimelineHeightChanged).toHaveBeenCalled();
+                    done();
+                }, 150); // higher timeout than in MessageComposer::onAction
```
Comment (kerryarchibald) [inline test/components/views/rooms/MessageComposer-test.tsx:105]: Not very nice to add all this delay to the tests, could you use fake timers and advance time instead?

**Review by kerryarchibald — APPROVED** [review]
LGTM! Left a few nitpick comments

File: src/settings/Settings.tsx (hunk 2 above)
Comment (t3chguy) [reply src/settings/Settings.tsx:105]: ++ for real enum, also it think as per the new code style this shouldn't be all caps

**Pull request merged**